fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchRevIncludeTest.java

```diff
@@ -0,0 +1,774 @@
+            Response response   = target.path("Procedure/" + procedure1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (procedure2Id != null) {
+            Response response   = target.path("Procedure/" + procedure2Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (procedure3Id != null) {
+            Response response   = target.path("Procedure/" + procedure3Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (procedure4Id != null) {
+            Response response   = target.path("Procedure/" + procedure4Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (procedure5Id != null) {
+            Response response   = target.path("Procedure/" + procedure5Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (organization1Id != null) {
+            Response response   = target.path("Organization/" + organization1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (practitioner1Id != null) {
+            Response response   = target.path("Practitioner/" + practitioner1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        if (encounter1Id != null) {
+            Response response   = target.path("Encounter/" + encounter1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+        for (String nutritionOrderId : nutritionOrderIds) {
+            Response response   = target.path("NutritionOrder/" + nutritionOrderId).request(FHIRMediaType.APPLICATION_FHIR_JSON).delete();
+            assertResponse(response, Response.Status.OK.getStatusCode());
+        }
+    }
+
+    @Test(groups = { "server-search-revinclude" })
+    public void testSearchRevIncludeWithSortError() {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("Patient")
+                .queryParam("_revinclude", "Procedure:patient")
+                .queryParam("_sort", "active")
+                .request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .get();
+        assertResponse(response, Response.Status.BAD_REQUEST.getStatusCode());
+        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class),
+                "_sort search result parameter not supported with _include or _revinclude.");
```

Comment:
in my opinion, this is an artificial limitation.  that said, its not really related to this PR and so I might open a separate issue on it (if anything)